libass/ass_filesystem.c

```diff
@@ -50,6 +50,43 @@ static bool alloc_path(ASS_Dir *dir, size_t size)
 }
 
 
+#if HAVE_FSTAT || _WIN32
+// Microsoft docs require types.h to be included before stat.h
+// IN POSIX just stat.h suffices
+#if _WIN32
```

Comment:
Minor/optional, but I’d prefer `#ifdef` and `#if ... defined(_WIN32)`. This is what we use elsewhere and seems intuitively safer, although everything I can find does say it’s specifically defined with a value of `1` rather than an empty one.

Reply:
done